crypto/ct/ct_prn.c

```diff
@@ -96,6 +96,27 @@ static void timestamp_print(uint64_t timestamp, BIO *out)
     ASN1_GENERALIZEDTIME_free(gen);
 }
 
+const char *SCT_validation_status_string(const SCT *sct)
+{
+
+    switch (SCT_get_validation_status(sct)) {
+    default:
```

Comment:
Better to remove the default case and instead have `return "unknown status"` after the switch. That way, the compiler can warn that not all enum values are handled in this switch when someone adds a new one.